jujugui/static/gui/src/app/app.js

```diff
@@ -1780,6 +1784,7 @@ YUI.add('juju-gui', function(Y) {
           getEntity={getEntity}
           getFile={charmstore.getFile.bind(charmstore)}
           getDiagramURL={charmstore.getDiagramURL.bind(charmstore)}
+          getModelName={getModelName}
```

Comment:
So then this here could just be `modelName={this.env.get('environmentName')}`. This is how we do it in the deployment flow.